File: Bundles/Raven.Bundles.Replication/Data/ReplicationDocument.cs
using System.Collections.Generic;

namespace Raven.Bundles.Replication.Data
{
    public class ReplicationDocument
    {
        public List<ReplicationDestination> Destinations { get; set; }

        public string Id { get; set; }

        public ReplicationDocument()
        {
            Id = "Raven/Replication/Destinations";
            Destinations = new List<ReplicationDestination>();
        }
    }
}

File: Raven.Client.Java/src/main/java/net/ravendb/abstractions/replication/ReplicationDocument.java
package net.ravendb.abstractions.replication;

import java.util.ArrayList;
import java.util.List;


public class ReplicationDocument {

  private List<ReplicationDestination> destinations;
  private String id;
  private String source;

  public ReplicationDocument() {
    id = "Raven/Replication/Destinations";
    destinations = new ArrayList<>();
  }

  public List<ReplicationDestination> getDestinations() {
    return destinations;
  }

  public String getId() {
    return id;
  }

  public String getSource() {
    return source;
  }

  public void setDestinations(List<ReplicationDestination> destinations) {
    this.destinations = destinations;
  }

  public void setId(String id) {
    this.id = id;
  }

  public void setSource(String source) {
    this.source = source;
  }


}
